Makefile

```diff
@@ -50,7 +50,7 @@ update-vendor:
 	hack/update-vendor.sh
 
 .PHONY: unit-test
-unit-test: update-vendor
+unit-test: autogen
```

Comment:
This is now needed as the testing in crossnodepreemption now needs the open API stuff to be fully generated. Otherwise, the test fails:

```
?   	sigs.k8s.io/scheduler-plugins/pkg/apis/config/scheme	[no test files]
?   	sigs.k8s.io/scheduler-plugins/pkg/apis/config/v1beta1	[no test files]
ok  	sigs.k8s.io/scheduler-plugins/pkg/coscheduling	0.756s
# k8s.io/kubernetes/cmd/kube-apiserver/app
vendor/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:467:70: undefined: "k8s.io/kubernetes/pkg/generated/openapi".GetOpenAPIDefinitions
FAIL	sigs.k8s.io/scheduler-plugins/pkg/crossnodepreemption [build failed]
ok  	sigs.k8s.io/scheduler-plugins/pkg/noderesources	0.187s
ok  	sigs.k8s.io/scheduler-plugins/pkg/qos	0.150s
FAIL
make: *** [Makefile:54: unit-test] Error 2
```